metacat-main/src/test/groovy/com/netflix/metacat/main/services/impl/TableServiceImplSpec.groovy

```diff
@@ -0,0 +1,112 @@
+/*
+ *
+ * Copyright 2018 Netflix, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.main.services.impl
+
+import com.netflix.metacat.common.QualifiedName
+import com.netflix.metacat.common.server.connectors.ConnectorTableService
+import com.netflix.metacat.common.server.converter.ConverterUtil
+import com.netflix.metacat.common.server.events.MetacatEventBus
+import com.netflix.metacat.common.server.properties.Config
+import com.netflix.metacat.common.server.usermetadata.TagService
+import com.netflix.metacat.common.server.usermetadata.UserMetadataService
+import com.netflix.metacat.main.manager.ConnectorManager
+import com.netflix.metacat.main.services.DatabaseService
+import com.netflix.metacat.main.services.TableService
+import com.netflix.metacat.testdata.provider.DataDtoProvider
+import com.netflix.spectator.api.NoopRegistry
+import spock.lang.Specification
+
+/**
+ * Tests for the TableServiceImpl.
+ *
+ * @author amajumdar
+ * @since 1.2.0
+ */
```

Comment:
I am not sure if this should be 1.2.0 or 1.1.1. Some codes are marked as 1.1.1 previously, and we should make them consistent as they are from the same change,

Reply:
Yes, all these changes will go as 1.2.0. I will change where ever it says 1.1.1